src/popup/router/pages/Notifications.vue

```diff
@@ -3,7 +3,11 @@
     <ae-list class="noti-list">
       <ae-list-item fill="neutral" v-for="(noti, i) in notifications" :key="i" class="noti">
         <img src="../../../icons/icon_48.png" />
-        <span> {{ noti.content }} </span>
+        <span>
+          <Component :is="noti.route ? 'RouterLink' : 'span'" :to="noti.route">
+            {{ noti.content }}
+          </Component>
+        </span>
```

Comment:
```suggestion
        <Component :is="noti.route ? 'RouterLink' : 'span'" :to="noti.route">
          {{ noti.content }}
        </Component>
```
Aha, I forgot to close Component tag, can we try this?